docs/dev/adding_instruments.rst

```diff
@@ -187,92 +227,78 @@ In addition, any entries in ``**kwargs**`` take precedence, so if they need to,
 
 For many instruments, the simple way presented first is enough, but in case you have a more complex arrangement to implement, see whether advanced_communication_protocols_ fits your bill. If, for some exotic reason, you need a special connection type, which you cannot model with PyVISA, you can write your own Adapter.
 
+.. _writing_properties:
 
 Writing properties
 ==================
 
-In PyMeasure, `Python properties`_ are the preferred method for dealing with variables that are read or set. 
-PyMeasure comes with two convenience functions for making properties for classes. 
-The :func:`Instrument.measurement <pymeasure.instruments.Instrument.measurement>` function returns a property that issues a GPIB/SCPI requests when the value is used. 
-For example, if our "Extreme 5000" has the :code:`*IDN?` command we can write the following property to be added above the :code:`def __init__` line in our above example class, or added to the class after the fact as in the code here:
+In PyMeasure, `Python properties`_ are the preferred method for dealing with variables that are read or set.
+
+The property factories
+**********************
+PyMeasure comes with three central convenience factory functions for making properties for classes: :func:`Instrument.control <pymeasure.instruments.Instrument.control>`, :func:`Instrument.measurement <pymeasure.instruments.Instrument.measurement>` and :func:`Instrument.setting <pymeasure.instruments.Instrument.setting>`.
```

Comment:
In English, you have a comma before "and" in an enumeration.
```suggestion
PyMeasure comes with three central convenience factory functions for making properties for classes: :func:`Instrument.control <pymeasure.instruments.Instrument.control>`, :func:`Instrument.measurement <pymeasure.instruments.Instrument.measurement>`, and :func:`Instrument.setting <pymeasure.instruments.Instrument.setting>`.
```

Reply:
Well, in [Oxford](https://en.wikipedia.org/wiki/Serial_comma), at least. :wink: 
(for the record, I also use the Oxford comma, just missed that one)